lamp_haproxy/aws/roles/nagios/tasks/main.yml

```diff
@@ -15,22 +17,31 @@
   notify: restart httpd
 
 - name: create nagios config dir
-  file: path=/etc/nagios/ansible-managed state=directory
+  file:
+    path: /etc/nagios/ansible-managed
+    state: directory
 
 - name: configure nagios
-  copy: src=nagios.cfg dest=/etc/nagios/nagios.cfg
+  copy:
+    src: nagios.cfg
+    dest: /etc/nagios/nagios.cfg
   notify: restart nagios
 
 - name: configure localhost monitoring
-  copy: src=localhost.cfg dest=/etc/nagios/objects/localhost.cfg
+  copy:
+    src: localhost.cfg
+    dest: /etc/nagios/objects/localhost.cfg
   notify: restart nagios
 
 - name: configure nagios services
-  copy: src=ansible-managed-services.cfg dest=/etc/nagios/
+  copy:
+    src: ansible-managed-services.cfg
+    dest: /etc/nagios/
 
 - name: create the nagios object files
-  template: src={{ item + ".j2" }}
-            dest=/etc/nagios/ansible-managed/{{ item }}
+  template:
+    src: "{{ item + .j2 }}"
```

Comment:
This will fail. `.j2` must be wrapped with single quotes, otherwise it's a syntax error.

Reply:
Whoops, goot catch on that.  I'll fix it up shortly.